Sources/VaporToolbox/Build.swift

```diff
@@ -67,14 +67,14 @@ public final class Build: Command {
         do {
             _ = try console.backgroundExecute(program: "swift", arguments: command)
             buildBar.finish()
-        } catch ConsoleError.backgroundExecute(let code, let error) {
+        } catch ConsoleError.backgroundExecute(let code, let error, _) {
             buildBar.fail()
             console.print()
```

Comment:
The third variable in the error enum should be called `let output` and we should print it here. (This build script was actually the whole reason I added it)

The output should look like:

```sh
Building [Failed]

Command:
swift build

Output:
blah
blah
blah

Error (1):
blah blah
blah

Toolchain:
blah
```